docs/docusaurus/docs/cloud/expectation_suites/manage_expectation_suites.md

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_label: 'Manage Expectation Suites'
+title: 'Manage Expectation Suites'
+description: Create and manage Expectation Suites in GX Cloud.
+---
+
+Expectation Suites contain multiple Expectations for a single Data Asset. Like Expectations, they can help you better understand your data and help you improve data quality. A default Expectation Suite is created when you create a Data Asset. 
+
+To learn more about Expectation Suites, see [Expectation Suites](../../terms/expectation_suite.md).
+
+## Prerequisites
+
+- You have [set up your environment](../set_up_gx_cloud.md) and the GX Agent is running. 
+
+- You have a [Data Asset](/docs/cloud/data_assets/manage_data_assets#create-a-data-asset).
+
+## Automatically create an Expectation Suite that tests for missing data
+
+Automatically create an Expectation Suite that you can use to determine if your Data Asset contains missing data (null values). Creating Expectation Suites automatically saves you writing and then running the same code for each column in your Data Asset.
+
+1. In GX Cloud, click **Data Assets** and select a Data Asset in the **Data Assets** list.
+
+2. Click the **Expectations** tab.
+
+3. Click **Create New Suite** in the **Expectation Suites** pane.
+
+4. Click **Missingness** and then enter a name for the Expectation Suite in the **Suite name** field.
+
+5. Click **Generate Expectations**. 
+
+    A new Expectation Suite appears in the Expectation Suites pane.
```

Comment:
Could we add some kind of note here about needing to wait for it to complete? It can take a couple of minutes, and so it would be good for users to have the expectation that they'll need to wait.

Reply:
@talagluck I added the requested edit.